assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -192,14 +192,18 @@
 			}
 
 			// Fix placeholder placement for Container with `flex-direction: row`.
-			if ( ! $( currentElement ).hasClass( 'elementor-first-add' ) && $( currentElement ).parents( '.e-container--placeholder-row' ).length ) {
+			const isRowContainer = $( currentElement ).parents( '.e-container--placeholder-row' ).length,
+				isFirstInsert = $( currentElement ).hasClass( 'elementor-first-add' );
```

Comment:
Save the element to a const instead of query the DOM many times

Reply:
Done